src/ConfigEditor.tsx

```diff
@@ -1,81 +1,73 @@
-import React, { ChangeEvent, PureComponent } from 'react';
+import React, { ChangeEvent } from 'react';
```

Comment:
```suggestion
import * as React from 'react';
```
As I said earlier on a call with you, we prefer to import React like the following in order to prevent to import the whole React package

Reply:
can you give me more informations about "to prevent to import the whole React package" please ?